generator/AWSPSGeneratorLib/HelpMaterials/Examples/IAM/Remove-IAMUserTag.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>Remove-IAMUserTag -UserName joe -TagKey "abac"</code>
+    <description>This example removes the tag from the user named "joe" with tag key as "abac".To remove multiple tags, provide a comma separted tag keys list.</description>
```

Comment:
It would be better to simply show the usage of the list instead of telling users that they can use one.